tests/e2e/specs/in-game/layout.spec.js

```diff
@@ -7,16 +7,16 @@ describe('Game View Layout', () => {
     cy.setupGameAsP0();
   });
 
-  it.skip('Hides navbar on gameview page', () => {
-    cy.get('[data-cy=nav-drawer]').should('not.be.visible');
+  it('Hides navbar on gameview page', () => {
+    cy.get('[data-cy=nav-drawer]').should('not.exist');
```

Comment:
Switched the method here since the sidebar isn't actually ever added to the gameview now.